shared/teams/channel/rows/member-row.tsx

```diff
@@ -87,7 +87,17 @@ const ChannelMemberRow = (props: Props) => {
     username &&
     dispatch(RouteTreeGen.createNavigateAppend({path: [{props: {teamID, username}, selected: 'teamMember'}]}))
   const onOpenProfile = () => username && dispatch(ProfileGen.createShowUserProfile({username}))
-  const onRemoveFromChannel = () => console.log('onRemoveFromChannel not yet implemented')
+  const onRemoveFromChannel = () =>
+    dispatch(
+      RouteTreeGen.createNavigateAppend({
+        path: [
+          {
+            props: {conversationIDKey, members: [username], teamID},
+            selected: 'teamReallyRemoveChannelMember',
```

Comment:
This is rough on mobile
<img width="200" alt="image" src="https://user-images.githubusercontent.com/11968340/79879456-b0b58d80-83bc-11ea-9133-ab799280cfe4.png">
Also shouldn't appear for `#general`

Reply:
I made a ticket to fix this and do the thing design wanted with all these icon in the corner of thing components, Y2K-1829. Will remove on general though, good point